integration/integrate-with-email.adoc

```diff
@@ -1,4 +1,4 @@
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: ASSEMBLY
 [id="integrate-using-email"]
 = Integrating with email
```

Comment:
Noticed that the assembly title is lowercase "email" but it's "Email" in the TOC. Maybe update the TOC to match this?